docs/designers-developers/developers/slotfills/plugin-document-setting-panel.md

```diff
@@ -0,0 +1,25 @@
+# PluginDocumentSettingPanel
+
+This SlotFill allows adding items below the Status & Availability panel in the Document Sidebar.
+
+## Available Props
+* __name__ `string`: A string identifying the panel.
+* __className__ `string`: An optional class name added to the sidebar body.
+* __title__ `string`: Title displayed at the top of the sidebar.
+* __icon__ `(string|Element)`: The [Dashicon](https://developer.wordpress.org/resource/dashicons/) icon slug string, or an SVG WP element.
+
+## Example
+```js
+const { registerPlugin } = wp.plugins;
+const { PluginDocumentSettingPanel } = wp.editPost;
+
+const PluginDocumentSettingPanelDemo = () => (
+	<PluginDocumentSettingPanel
+		name="custom-panel"
+		title="Custom Panel"
+		className="custom-panel"
+	>
+		Custom Panel Contents
+	</PluginDocumentSettingPanel>
+);
+registerPlugin( 'plugin-document-setting-panel-demo', { render: PluginDocumentSettingPanelDemo, icon: 'palmtree' } );
```

Comment:
Can you close this code block? I'm not actually sure if the dot-org markdown parser would be able to correctly handle an unclosed code block.

Reply:
Nice catch! I've updated.